scripts/commands/updateApiDocs.ts

```diff
@@ -60,6 +61,14 @@ const readArgs = (): Arguments => {
       description:
         "Rather than downloading the artifact from Box, reuse what is already downloaded. This can save time, but it risks using an outdated version of the docs.",
     })
+    .option("sphinx-artifact-folder", {
+      alias: "sphinx-output",
```

Comment:
If you want an alias, I'd rather it be something short like `-a` for `artifact`. Right now, the two names feel quite similar and like we couldn't decide on the name

Reply:
I don't really mind - this was mostly borne out of me not being to work out to have `yargs` have a command-line argument that differed from the name of the associated property in the object.  I wanted to _just_ have the command-line argument be `--sphinx-output`, but the internal details of the code want that property to be `sphinxArtifactFolder` for more consistency, which felt not-so-descriptive to me outside the context of the internal code.

Anyway, I don't mind much about the naming as long as the option exists - if you're rather I just remove the alias entirely, that's fine.

Reply:
Either remove the alias or use `-a` (or you can suggest something else). I like having the short alias to make it easier to use this script

Reply:
Done in 8c6434f5d1e14843f57cd8108934d510f959bb27.